packages/components/src/toolbar/stories/index.js

```diff
@@ -20,22 +27,30 @@ export const _default = () => {
 		// id is required for server side rendering
 		<Toolbar __experimentalAccessibilityLabel="Options" id="options-toolbar">
 			<ToolbarGroup>
-				<ToolbarButton icon="editor-paragraph" title="Paragraph" />
+				<ToolbarButton icon="editor-paragraph" label="Paragraph" />
+			</ToolbarGroup>
+			<ToolbarGroup>
```

Comment:
Is there any difference between this and:

![Screen Shot 2020-01-03 at 12 09 57](https://user-images.githubusercontent.com/699132/71720424-fc4e2680-2e21-11ea-9cb6-4486d1fc8200.png)

It looks like we have the collapsed group case covered :)

Reply:
There are some cases in Gutenberg where we're using `DropdownMenu` directly instead `<ToolbarGroup isCollapsed>`. For example: https://github.com/WordPress/gutenberg/blob/2c1e3be7d1cec76176db6cccee1c6add93cc3e4d/packages/block-editor/src/components/block-settings-menu/index.js#L59-L66

I'm not sure how easy is it to refactor those edge cases to use `<ToolbarGroup isCollapsed>`, so I just wanted to make sure  `ToolbarItem` can be used.

Reply:
Cool, it all makes sense. I think I mentioned it already, I just wanted to ensure you are aware of the current state of the art :)